<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayoutViewer</class>
 <widget class="QDialog" name="LayoutViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelImage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>33</y>
     <width>581</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClose">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>99</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttomZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zoom-in.png</normaloff>:/images/zoom-in.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zoom-out.png</normaloff>:/images/zoom-out.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAboutLayout">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>About Layout...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewNormal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal View</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/keyboard_layout.png</normaloff>:/images/keyboard_layout.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="viewAltGr">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AltGr View</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/keyboard_layout.png</normaloff>:/images/keyboard_layout.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
